CHANGELOG.adoc

```diff
@@ -61,6 +61,7 @@ A reset of the database is needed.
 - https://github.com/eclipse-syson/syson/issues/371[#371] [diagrams] Handle the ForkNode concept in actions body of diagrams
 - https://github.com/eclipse-syson/syson/issues/381[#381] [diagrams] Handle the MergeNode concept in actions body of diagrams
 - https://github.com/eclipse-syson/syson/issues/389[#389] [diagrams] Handle the DecisionNode concept in actions body of diagrams
+- https://github.com/eclipse-syson/syson/issues/391[#391] [diagrams] AcceptAction should be available inside an Action body
```

Comment:
If it is a new feature, then you should not use "should" in your statement. "should" is rather for bug fixes.

Reply:
Or if it is a bug fix, please move it to the appropriate section, and update the message with something like : `Fix an issue where the AcceptAction were not available inside an Action body`

Reply:
Agree. I've changed the issue headline to `AcceptAction is now available inside an Action body`.

Reply:
Actually, this is not an issue since it is part of the action flow compartment development.

Reply:
Ok